src/main/Core/IniFileParser/Contract/IniFileParser.ts

```diff
@@ -4,14 +4,16 @@
 export interface IniFileParser {
     /**
      * Parses the given ini file content and returns the result as an object.
-     * @param fileString The ini file content, e.g.:
+     * @param {string} fileString The ini file content, e.g.:
```

Comment:
I would remove the types from these coments, as the types are already defined via TypeScript.